Bottom: f36ac99c2b81360c21d3a88ffbf8d6ee9369d66e
Top:    ccc5c0493ee56b604cdddff93381d25b6e4269d4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-22 21:43:43 +0000

Refresh of free_atexit-c-add-leaky_calloc

---

diff --git a/src/free_atexit.c b/src/free_atexit.c
index d5c034e..3795c83 100644
--- a/src/free_atexit.c
+++ b/src/free_atexit.c
@@ -58,10 +58,10 @@ void *leaky_malloc (size_t size)
 
 void *leaky_calloc (size_t nmemb, size_t size)
 {
-	size_t size_ = size * nmemb;
-  void *new_memory = leaky_malloc(size_);
+  size_t size_ = size * nmemb;
+  void *new_memory = leaky_malloc (size_);
 
-	memset(new_memory, 0, size_);
+  memset (new_memory, 0, size_);
   return new_memory;
 }
